#!/bin/sh /usr/share/dpatch/dpatch-run
## 02-fix-xcscope-el-typo.dpatch
##
## DP: Fixes a typo in contrib/xcscope/xcscope.el
## DP: (Closes: #438170)

--- cscope-15.6/contrib/xcscope/xcscope.el.orig	2007-08-16 13:38:28.000000000 +0200
+++ cscope-15.6/contrib/xcscope/xcscope.el	2007-08-16 13:38:38.000000000 +0200
@@ -114,7 +114,7 @@
 ;;	(define-key global-map [(control f11)] 'cscope-prev-symbol)
 ;;	(define-key global-map [(control f12)] 'cscope-prev-file)
 ;;      (define-key global-map [(meta f9)]  'cscope-display-buffer)
-;;      (defin-ekey global-map [(meta f10)] 'cscope-display-buffer-toggle)
+;;      (define-key global-map [(meta f10)] 'cscope-display-buffer-toggle)
 ;;
 ;; 6. Restart (X)Emacs.  That's it.
 ;;
